### PR TITLE
[homekit] migrate to bnd

### PR DESCRIPTION
**Pull request by J-N-K**
Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

**Comment by J-N-K** [comment]
Also removes unneeded poms

**Comment by davidgraeff** [comment]
I guess the poms/bnd/pom.xml should be merged into the root pom.xml

**Comment by J-N-K** [comment]
Yes, but that requires changes in all bundes-poms, therefore I left that for a follow-up PR

File: bundles/org.openhab.io.homekit/.project [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>org.openhab.io.homekit</name>
+	<name>org.openhab.binding.allplay</name>
```
Comment (Hilbrand) [inline bundles/org.openhab.io.homekit/.project]:
```suggestion
	<name>org.openhab..io.homekit</name>
```

**Comment by J-N-K** [comment]
Fixed.

**Comment by davidgraeff** [comment]
Travis fails because of modbus:
```
[ERROR] Failed to execute goal org.apache.karaf.tooling:karaf-maven-plugin:4.2.4:verify (karaf-feature-verification) on project org.openhab.addons.features.karaf.openhab-addons: Feature resolution failed for [openhab-binding-modbus/2.5.0.SNAPSHOT]
[ERROR] Message: org.apache.karaf.features.internal.util.MultiException: Error:
[ERROR] /var/jenkins_home/workspace/PR-openHAB2-Addons/bundles/org.openhab.binding.modbus/target/classes (Is a directory)
```

**Pull request merged**